Phonebook.Frontend/src/app/pages/settings/settings.component.ts

```diff
@@ -47,6 +52,45 @@ export class SettingsComponent implements OnInit, OnDestroy {
     this.layoutValue = layout;
   }
 
+  public getThemeName(theme: Theme) {
+    switch (theme) {
+      case Theme.blue_light_theme: {
+        return this.i18n({
+          value: 'Blue Light Theme',
+          description: 'Color Theme Option: Blue Light',
+          id: 'SettingsComponentColorThemeBlueLight',
+          meaning: 'SettingsComponent'
+        });
+      }
+      case Theme.blue_dark_theme: {
+        return this.i18n({
+          value: 'Blue Dark Theme',
+          description: 'Color Theme Option: Blue Dark',
+          id: 'SettingsComponentColorThemeBlueDark',
+          meaning: 'SettingsComponent'
+        });
+      }
+      case Theme.magenta_light_theme: {
+        return this.i18n({
+          value: 'Magenta Light Theme',
+          description: 'Color Theme Option: Magenta Light',
+          id: 'SettingsComponentColorThemeMagentaLight',
+          meaning: 'SettingsComponent'
+        });
+      }
+      case Theme.magenta_dark_theme: {
+        return this.i18n({
+          value: 'Magenta Dark Theme',
+          description: 'Color Theme Option: Magenta Dark',
+          id: 'SettingsComponentColorThemeMagentaDark',
+          meaning: 'SettingsComponent'
+        });
+      }
+      default: 
+        throw Error('Translation for theme ${theme} does not exists.');
```

Comment:
Das wird mit den `'` nicht funktionieren da brauchst du ``` ` ```
```suggestion
        throw Error(`Translation for theme ${theme} does not exists.`);
```